.devcontainer/post_start.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+git config --global --add safe.directory $1
```

Comment:
https://github.com/microsoft/vscode-remote-release/issues/7628

Reply:
The fix in devcontainer pre-release mentioned in https://github.com/microsoft/vscode-remote-release/issues/7628#issuecomment-1406157351 did not work for me